common/static/common/templates/discussion/templates.underscore

```diff
@@ -775,24 +838,28 @@
             <%- gettext("Post type") %>
         </span>
     </legend>
-    <div class="field-help" id="field_help_post_type">
+    <!-- <div class="field-help" id="field_help_post_type">
         <%- gettext("Questions raise issues that need answers. Discussions share ideas and start conversations. (Required)") %>
-    </div>
+    </div>-->
```

Comment:
Will this code be used later or is this an unnecessary comment.

Reply:
I think this will be used later